data/json/furniture_and_terrain/furniture-decorative.json

```diff
@@ -152,6 +152,28 @@
       "items": [ { "item": "rock", "count": [ 1, 6 ] }, { "item": "rebar", "count": [ 1, 2 ] } ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_gazing_ball",
+    "name": "gazng ball",
+    "description": "A mirrored glass globe mounted on a cement pedastal.  It warps your reflection when you look into it",
+    "symbol": "o",
+    "color": "white",
+    "move_cost_mod": -1,
+    "required_str": 10,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
```

Comment:
Does PLACE_ITEM flag really fits here? Based on the description, there is no place for items there.